test/fuzz/lua/test_engine.lua

```diff
@@ -500,12 +621,23 @@ local function format_op(space, space_format)
     space:format(space_format)
 end
 
+local function insert_arrow_op(space, batch)
+    arrow.box_insert_arrow(space.id, batch)
+end
+
 local function setup(engine_name, space_id_func, test_dir, verbose)
     log.info('SETUP')
     assert(engine_name == 'memtx' or
-           engine_name == 'vinyl')
+           engine_name == 'vinyl' or
+           engine_name == 'memcs')
     -- Configuration reference (box.cfg),
     -- https://www.tarantool.io/en/doc/latest/reference/configuration/
+
+    -- vinyl_page_size must be in range (0, vinyl_range_size].
```

Comment:
please move comment under the line `local vinyl_page_size = ...`

Reply:
Done.